samples/MegaDrone/src/main/cpp/Synth.h

```diff
@@ -18,11 +18,11 @@
 #define MEGADRONE_SYNTH_H
 
 #include <array>
-#include <shared/TappableAudioSource.h>
+#include <TappableAudioSource.h>
 
-#include "shared/Oscillator.h"
-#include "shared/Mixer.h"
-#include "shared/MonoToStereo.h"
+#include <Oscillator.h>
```

Comment:
Are these synth tools still shared?

Reply:
Yes they are shared, but since we are including the shared directory (instead of including the entire samples directory), they lack the shared directory prefix.